lib/pure/os.nim

```diff
@@ -447,6 +448,8 @@ proc getFilePermissions*(filename: string): set[FilePermission] {.
     if (a.st_mode and S_IROTH) != 0'i32: result.incl(fpOthersRead)
     if (a.st_mode and S_IWOTH) != 0'i32: result.incl(fpOthersWrite)
     if (a.st_mode and S_IXOTH) != 0'i32: result.incl(fpOthersExec)
+
```

Comment:
Maybe remove this space to be consistent with other if's?

Reply:
To me it looks like the permissionsare grouped by User, Group and Other. I don't know if Sticky Bit fits into those categories so I added the newline as if it were in a separate category. Same with the checkandincludemode further down.

Reply:
Ah, sorry, didn't see the whole thing